include/nbl/builtin/glsl/ext/DepthPyramidGenerator/depth_pyramid_generator.comp

```diff
@@ -0,0 +1,50 @@
+#version 430 core
+
+layout(local_size_x = WORKGROUP_X_AND_Y_SIZE, local_size_y = WORKGROUP_X_AND_Y_SIZE) in;

```

Comment:
its easier to make this work as SIZEx1x1 because once you refactor this whole thing, you'll have to use standardized defines such as `NBL_GLSL_WORKGROUP_SIZE`